### PR TITLE
feat: product reviews accordion [SV-29]

### PR DESCRIPTION
**Pull request by FRSgit**
### 🔗 Linked issue

https://vsf.atlassian.net/browse/SV-29

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/web/components/ProductAccordion/ProductAccordion.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      <UiReview v-for="review in productReviews" :key="review.id" :review="review" class="mb-4" />
+    </UiAccordionItem>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import type { SfProduct } from '@vue-storefront/unified-data-model';
+
+const props = defineProps<{
+  product: SfProduct;
```
[the remark this answers is not in the document]
Comment (FRSgit) [reply apps/web/components/ProductAccordion/ProductAccordion.vue]: 114692684beed5ae7a766a45ad22dd3f2a02b21b

File: apps/web/components/ProductAccordion/ProductAccordion.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <UiAccordionItem
+      summary-class="md:rounded-md w-full hover:bg-neutral-100 py-2 pl-4 pr-3 flex justify-between items-center"
+    >
+      <template #summary>
+        <h2 className="font-bold font-headings text-lg leading-6 md:text-2xl">
```
[the remark this answers is not in the document]
Comment (FRSgit) [reply apps/web/components/ProductAccordion/ProductAccordion.vue]: c1667481932df632a940373309a5a2b1e5be9a5e

File: apps/web/components/ProductAccordion/ProductAccordion.vue [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        </h2>
+      </template>
+      <p>{{ product.description }}</p>
+    </UiAccordionItem>
+    <UiDivider class="my-4" />
+    <UiAccordionItem
+      summary-class="md:rounded-md w-full hover:bg-neutral-100 py-2 pl-4 pr-3 flex justify-between items-center"
+    >
+      <template #summary>
+        <h2 className="font-bold font-headings text-lg leading-6 md:text-2xl">
```
[the remark this answers is not in the document]
Comment (FRSgit) [reply apps/web/components/ProductAccordion/ProductAccordion.vue]: c1667481932df632a940373309a5a2b1e5be9a5e

File: apps/web/components/ui/AccordionItem/AccordionItem.vue [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <SfAccordionItem v-model="internalModelValue" :summary-class="summaryClass">
+    <template #summary>
+      <slot name="summary">
+        <p>{{ summary }}</p>
+      </slot>
+      <SfIconChevronLeft :class="['text-neutral-500', modelValue ? 'rotate-90' : '-rotate-90']" />
+    </template>
+    <div className="py-2 px-4">
```
[the remark this answers is not in the document]
Comment (FRSgit) [reply apps/web/components/ui/AccordionItem/AccordionItem.vue]: c1667481932df632a940373309a5a2b1e5be9a5e

File: apps/web/components/ui/Review/Review.vue [hunk 5]
```diff
@@ -0,0 +1,61 @@
+<template>
+  <article class="w-full p-4 border rounded-md">
+    <p class="pb-2 font-medium">{{ review.title }}</p>
+    <header class="flex flex-col pb-2 md:flex-row md:justify-between">
+      <span class="flex items-center pr-2 text-xs text-neutral-500">
+        <SfRating :value="(review.rating as number)" :max="5" size="xs" class="mr-2" />
```
[the remark this answers is not in the document]
Comment (FRSgit) [reply apps/web/components/ui/Review/Review.vue]: 07f6722d24d523626d2f42eb7411de19b84abacd

File: apps/web/components/ProductAccordion/ProductAccordion.vue [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+<script lang="ts" setup>
+import type { ProductAccordionPropsType } from '~/components/ProductAccordion/types';
+
+const props = defineProps<ProductAccordionPropsType>();
+
+const { product } = toRefs(props);
+
+const { data: productReviews, fetchProductReviews } = useProductReviews(product.value.slug);
+fetchProductReviews(product.value.slug);
```
Comment (dawid-ziobro) [inline apps/web/components/ProductAccordion/ProductAccordion.vue:36]:
```suggestion
await fetchProductReviews(product.value.slug);
```

**Review by dawid-ziobro — APPROVED** [review]

**Review by justyna-13 — APPROVED** [review]

**Review by dawid-ziobro — APPROVED** [review]

**Pull request merged**